python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -676,72 +665,29 @@ def addToList(_input_elements, _final_list, action=None):
 
         return
 
-def export(method):
-    """
-    Decorate a GangaObject method to be exported to the GPI
-    """
-    method.exported_method = True
-    return method
-
 
 class ObjectMetaclass(type):
     _descriptor = Descriptor
 
     def __init__(cls, name, bases, this_dict):
 
-        from Ganga.GPIDev.Base.Proxy import GPIProxyClassFactory, ProxyDataDescriptor, ProxyMethodDescriptor
+        from Ganga.GPIDev.Base.Proxy import GPIProxyClassFactory
 
         super(ObjectMetaclass, cls).__init__(name, bases, this_dict)
 
-        # ignore the 'abstract' base class
-        # FIXME: this mechanism should be based on explicit getName(cls) or alike
-        #if name == 'GangaObject':
-        #    return
-
-        #logger.debug("Metaclass.__init__: class %s name %s bases %s", cls, name, bases)
-
         # all Ganga classes must have (even empty) schema
-        if not hasattr(cls, '_schema') or cls._schema is None:
+        if cls._schema is None:
             cls._schema = Schema.Schema(None, None)
 
```

Comment:
Do you mean "passed a `GangaObject`"? If so, no. There are no cases (at present) where anything other than `GangaObject` has its metaclass set to `ObjectMetaclass`.